core/pom.xml

```diff
@@ -413,6 +413,16 @@
       <artifactId>hadoop-minikdc</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.bouncycastle</groupId>
+      <artifactId>bcprov-jdk15on</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.bouncycastle</groupId>
+      <artifactId>bcpkix-jdk15on</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Just a question. What happens for `hadoop-2.7` profile? Can we put this under `hadoop-3.2` profile?

Reply:
Oops forgot it. Yes I should put it under `hadoop-3.2`.